assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| REQM              | Fully | The requirements are in user stories in the backlog and are the responsibility of the product owner to determine and formulate.            |
+| SAM               | Not at all | Scrum only prescribes some practices on how a team should develop software and does not have information on dealing with a supplier.             |
+| **Level 3**                                 |
+| DAR               | Not at all | Scrum does not have any formal evaluation method which can be used analyze possible decisions or alternatives            |
+| IPM               | Partially | There are guidelines in place to manage when stakeholders can interact with its team but for the most part this up to the product owner            |
+| OPD               | Not at all | This is left entirely up to the wishes of the development team            |
+| OPF               | Partially | Scrum's retrospective allows for teams to come up with improvements to the process.             |
+| OT                | Not at all | Identifying training needs of an organization and assessing its effectiveness is not covered in Scrum. |
+| PI                | Partially | Scrum states that at the end of each sprint a product should be developed, tested and ready for production. It does not cover how this integration should be achieved. |
+| RD                | Partially | Scrum addresses this through the use of user stories in the Product Backlog and Sprint Backlog. Its responsibility however is almost entirely delegated to the Product Owner and Scrum does not offer guidelines in how to collect and coördinate stakleholder needs and translate them in requirements.           |
+| RSKM              | Not at all | Identifying potential risks and how to deal with them is not covered in Scrum.      |
+| VAL               | Partially | A form of validation is done through the Sprint Review in which the prodcut is demonstrated to the stakeholders. Other aspects of validation such as training, manufacturing and maintenance are not covered by Scrum. |
+| VER               | Partially | Verification is done in-sprint in the form of testing activities and during the Sprint Review in which the product is demonstrated to its stakeholders. Scrum does not state what kind of methods, such as peer reviews, test methods, etc. should be applied.            |
+| **Level 4**                              |
+| OPP               | Partially | Organizational process performance is covered in a Sprint Retrospective. This retrospective has however little quantitive properties. It is based soley on subjective input of inidividual team members and does not deal with objective measurements. |
+| QPM               | Not at all | Quantitive management of the team's quality and process perfromance is not covered in Scrum.           |
```

Comment:
the team's quality is not a very clear term to me. Is it the quality of the team itself, or the work they produce? Also performance is misspelled